src/lua/zencode_ethereum.lua

```diff
@@ -151,7 +151,7 @@ function(obj)
   local content = have(obj)
   local tx = have'ethereum transaction'
   ZEN.assert(not tx.data or #tx.data == 0, "Cannot overwrite transaction data")
-  tx.data = ETH.make_storage_data(content)
+  tx.data = ETH.make_storage_data(ZEN.serialize(content))
```

Comment:
serialize is one way (irreversible) is that enough?

Reply:
Ah, it is irreversible, no it is not enough...

Reply:
then I guess it should be:
`O.to_base64( O.from_string( JSON.encode( content, 'base64' ) ) )`

Reply:
Thanks